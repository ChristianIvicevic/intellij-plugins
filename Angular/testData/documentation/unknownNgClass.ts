import { Component } from '@angular/core';
import {NgClass} from "@angular/common";

@Component({
  selector: 'app-ng-class',
  imports: [
    NgClass
  ],
  template: `
    <p
        [ngClass]="{
        'bar': true,
          'font-bold text-<caret>align': true,
        }"
    ></p>
  `,
  standalone: true
})
export class UnknownNgClass {

}
