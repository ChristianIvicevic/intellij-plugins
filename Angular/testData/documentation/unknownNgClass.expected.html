<div class='definition'><pre><span style="color:#660e7a;">[&quot;font-bold text-align&quot;]</span>: <span style="color:#000080;">boolean</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;unknownNgClass.ts</td></table>